src/applications/disability-benefits/2346/helpers/index.js

```diff
@@ -71,3 +72,20 @@ export function validateZIP(errors, zip) {
     );
   }
 }
+
+export const newAddressHider = formData => {
+  const address = get('newAddress', formData) || {};
+  // return true if living at view:militaryBaseInfo object
+  return Object.keys(address).every(entry => {
+    const value = address[entry] || '';
+    return typeof value === 'string' ? value === '' : true;
```

Comment:
`return typeof value !== 'string'`, although this function may need some love to ensure it's doing what's desired! 🙂

Reply:
Yeah I have no idea whats going on here - I will have to dig into this further